boot/freeldr/freeldr/arch/i386/i386bug.c

```diff
@@ -115,9 +116,11 @@ i386PrintExceptionText(ULONG TrapIndex, PKTRAP_FRAME TrapFrame, PKSPECIAL_REGIST
     i386_ScreenPosX = 0;
     i386_ScreenPosY = 0;
 
-    PrintText("An error occured in " VERSION "\n"
+    PrintText("Error in %s %s\n"
```

Comment:
```suggestion
    PrintText("An error occured in " VERSION " (Build " KERNEL_VERSION_BUILD_STR ")\n"
```

Reply:
Your suggestion would not show the arch. And I don't see at all what VERSION 3.2.0 is telling us actually, if we have 100000revisions and changes in freeldr, but it always stays 3.2.0 for 10years+.

![image](https://github.com/user-attachments/assets/72b7e538-ae95-4c02-a9b8-b31eb08b9fba)

Reply:
I would accept turning `"Error in %s %s\n"` into "FreeLdr %s %s\n" if that would make you happy also.

Reply:
Or `"Error in FreeLdr %s %s\n"` to make it more readable.
By the way, any reason why not including the build strings directly in the message, instead of using the %s stuff?

Reply:
I applied your suggestion with the stripping of %s %s. The final result looks like that (I prefer to have the static parts of that line as short as possible, because we do have only 80chars available per line, and I don't want to risk causing any line-overflow, in case we would increase some strings length slightly in the future)
![image](https://github.com/user-attachments/assets/1aab050f-d3f1-4b54-9c06-99ef65719560)

Keep in mind that Line2 also begins with the word "Error", so I don't think we will loose any expressiveness when performing like that, and we do even get the hint back, that it is the freeldr module which is generating that screen. Yes, that is obvious for us developers, but maybe not for 3rd party hobby-coders/-testers who might run into the screen.

Personally I do think in the long term it would make sense to strip the 3.2.0 version number from freeldr entirely and just stick with the ros isos version identification, because that one automatically increases when code is touched, which is unfortunately not the case for the old-handgroomed-way of doing things (or let's better say: way of never doing things). handgroomed does not scale here at all, because freeldr does even rely on some external components here and there. So it will change even if the module is not touched at all. Nobody could ever properly reflect such things in the old buildnr. It's just impossible for humans to oversee the complexity of whole ros.